### DISCUSSION TITLE
How to make multiple modal windows?

### DISCUSSION THREAD

**Discussion by DANPaRUS** [heroui-inc/heroui#2213] [Q&A]
I need to use modal window above other modal window. How I can make it?

**Comment by ali-shafi-hff** [comment]
modal over modal is not a good User Experience.
What you can do is disable the previous modal on opening the new Modal and on closing the 2nd modal enable the first one.

**Comment by wingkwong** [comment]
You can simply use two `useDisclosure` to control the modals.
